components/x-teaser/src/AlwaysShowTimestamp.jsx

```diff
@@ -0,0 +1,21 @@
+import { h } from '@financial-times/x-engine'
+import TimeStamp from './TimeStamp'
+import RelativeTime from './RelativeTime'
+import { differenceInCalendarDays } from 'date-fns'
+import { getDateOnly } from '../../x-teaser-timeline/src/lib/date'
```

Comment:
**issue**: to use `x-teaser-timeline` you'd need to add it as a dependency of `x-teaser` and refer to it by package name (since they're separate packages, you can't rely on relative paths). however, it doesn't feel right to have `x-teaser` depend on `x-teaser-timeline`; could this function be extracted into a common library, or is there maybe something we could use from `date-fns`?